src/usecase/tests/meal_ucase_test.go

```diff
@@ -0,0 +1,130 @@
+package tests
+
+import (
+	"github.com/Aiscom-LLC/meals-api/src/delivery"
+	"github.com/Aiscom-LLC/meals-api/src/delivery/middleware"
+	"github.com/Aiscom-LLC/meals-api/src/repository"
+	"github.com/appleboy/gofight/v2"
+	"github.com/buger/jsonparser"
+	"github.com/go-playground/assert/v2"
+	uuid "github.com/satori/go.uuid"
+	"net/http"
+	"testing"
+)
+
+func TestAddMeal(t *testing.T) {
+	r := gofight.New()
+
+	dishRepo := repository.NewDishRepo()
+	userRepo := repository.NewUserRepo()
+	categoryRepo := repository.NewCategoryRepo()
+	cateringRepo := repository.NewCateringRepo()
+	cateringResult, _ := cateringRepo.GetByKey("name", "Twiist")
+	cateringID := cateringResult.ID.String()
+	categoryResult, _ := categoryRepo.GetByKey("name", "гарнир", cateringID)
+	categoryID := categoryResult.ID.String()
+	userResult, _ := userRepo.GetByKey("email", "admin@meals.com")
+	dishResult, _, _ := dishRepo.GetByKey("name", "доширак", cateringID, categoryID)
+	jwt, _, _ := middleware.Passport().TokenGenerator(&middleware.UserID{userResult.ID.String()})
+	var dishID [1]string
+	dishID[0] = dishResult.ID.String()
```

Comment:
Please use short var declaration when it's possible, also don't use arrays, use slices (`[]int` instead of `[10]int`)

Reply:
`dishID := dishResult.ID.String()` ?

Reply:
Okay, I understood, do this instead then
`var dishIDs []string`
`dishIDs = append(dishIDs, dishResult.ID.String())`